providers/edge3/src/airflow/providers/edge3/plugins/www/src/components/RemoveQueueButton.tsx

```diff
@@ -0,0 +1,156 @@
+        title: "Queue Removed",
+        type: "success",
+      });
+      onClose();
+      setSelectedQueue("");
+    },
+  });
+
+  const handleRemoveQueue = () => {
+    if (!selectedQueue) {
+      onQueueUpdate({
+        description: "Please select a queue to remove.",
+        title: "Invalid Selection",
+        type: "error",
+      });
+      return;
+    }
+
+    removeQueueMutation.mutate({
+      queueName: selectedQueue,
+      workerName: worker.worker_name,
+    });
+  };
+
+  const availableQueues = worker.queues || [];
+  const isDisabled = availableQueues.length === 0;
+
+  return (
+    <>
+      <IconButton
+        size="sm"
+        variant="ghost"
+        onClick={onOpen}
+        aria-label="Remove Queue"
+        title="Remove Queue"
+        colorPalette="danger"
+        disabled={isDisabled}
+      >
+        <LuListMinus />
+      </IconButton>
+
+      <Dialog.Root onOpenChange={onClose} open={open} size="md">
+        <Portal>
+          <Dialog.Backdrop />
+          <Dialog.Positioner>
+            <Dialog.Content>
+              <Dialog.Header>
+                <Dialog.Title>Remove Queue from {worker.worker_name}</Dialog.Title>
+              </Dialog.Header>
+              <Dialog.Body>
+                <VStack gap={4} align="stretch">
+                  {availableQueues.length > 0 ? (
+                    <>
+                      <Text>Select a queue to remove from this worker:</Text>
+                      <VStack gap={2} align="stretch">
+                        <For each={availableQueues}>
+                          {(queue) => (
+                            <Button
+                              key={queue}
+                              variant={selectedQueue === queue ? "solid" : "outline"}
+                              colorPalette={selectedQueue === queue ? "blue" : "gray"}
+                              onClick={() => setSelectedQueue(queue)}
+                              justifyContent="flex-start"
+                            >
+                              {queue}
+                            </Button>
+                          )}
+                        </For>
+                      </VStack>
+                    </>
+                  ) : (
+                    <Text>This worker has no queues to remove.</Text>
+                  )}
+                </VStack>
+              </Dialog.Body>
+              <Dialog.Footer>
+                <Dialog.ActionTrigger asChild>
+                  <Button variant="outline">Cancel</Button>
+                </Dialog.ActionTrigger>
+                {availableQueues.length > 0 && (
+                  <Button
+                    onClick={handleRemoveQueue}
+                    colorPalette="danger"
+                    loading={removeQueueMutation.isPending}
+                    loadingText="Removing queue..."
+                    disabled={!selectedQueue}
+                  >
+                    <LuListMinus style={{ marginRight: "8px" }} />
```

Comment:
When I compare with the <DeleteDag /> there is also no extra margin - can we make it the same here?
```suggestion
                    <LuListMinus />
```

Reply:
Done!